appveyor.yml

```diff
@@ -26,11 +26,11 @@ environment:
       PYTHON_ARCH: "64"
 
     - PYTHON: "C:\\Python35"
```

Comment:
`C:\\Python35` needs to be changed to `C:\\Python37` otherwise the preinstalled folder is used IIRC.